src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -72,23 +72,31 @@ std::ostream& operator<<(std::ostream& os,
                          const Quadrature& quadrature) noexcept;
 /// \endcond
 
+namespace detail {
+constexpr size_t minimum_number_of_points(
+    const Basis basis, const Quadrature quadrature) noexcept {
+  // Silence compiler warnings.
+  (void)basis;
+  (void)quadrature;
```

Comment:
I can see how `(void)basis` might silence a warning, but why is `(void)quadrature` needed? The value is used in the if statement below...

Reply:
It does not. Once the other comments have been settled I'll remove.